server/lib/src/idm/server.rs

```diff
@@ -2095,6 +2119,11 @@ impl<'a> IdmServerProxyWriteTransaction<'a> {
 
     #[instrument(level = "debug", skip_all)]
     pub fn commit(mut self) -> Result<(), OperationError> {
+        if self.qs_write.get_changed_app() {
+            self.qs_write
+                .get_apprs_set()
+                .and_then(|apprs_set| self.apprs.reload(apprs_set))?;
```

Comment:
As above, we are trying to move away from abreviated names if possible.

Reply:
Fixed.